_posts/2021-09-18-p2pool-released.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Monero P2Pool is now live"
+summary: P2Pool decentralized pool for Monero mining is live on the the mainnet

```

Comment:
There is an added typo now (the the). I [suggested](https://github.com/monero-project/monero-site/pull/1835#discussion_r711699621):
```suggestion
summary: P2Pool, the decentralized pool for Monero mining, is live on mainnet

```

Reply:
Ack. Fixed.